.spelling

```diff
@@ -9,6 +9,8 @@
 1.x
 10s
 123456789012.my
+14.60ms.
```

Comment:
unrelated, but.. does this indicate that we should have a better spelling checker in place, @geeknoid ? I don't think we want to end up listing every possible duration in a file, right?

Reply:
It's lame, but that's the state of the art at this point for markdown spell checking :-(

Reply:
We can at lease decouple them from the number. The [best practice](https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/term-collections/units-of-measure-terms) is to have a space between the unit and the value.

Reply:
@rcaballeromx looking in the previously existing Distributed Tracing task I see the following:
`It took 22.99ms .` Where there is no space between the number and units but there is a space between the unit and the period.

Which of the following is preferred?:

`It took 14.60 ms.`
`It took 14.60ms .`
`It took 14.60 ms .`
`It took 14.60ms.`